website/docs/deploy/configuring-unleash.md

```diff
@@ -191,43 +197,40 @@ The logger interface with its `debug`, `info`, `warn` and `error` methods expect
-| `pool`                   | N/A (use the below variables)  |                | An object describing database pool options. With environment variables, use the options below directly.                                                                                  |
-| `pool.min`               | `DATABASE_POOL_MIN`             | 0              | The minimum number of connections in the connection pool.                                                                                                                                |
-| `pool.max`               | `DATABASE_POOL_MAX`             | 4              | The maximum number of connections in the connection pool.                                                                                                                                |
-| `pool.idleTimeoutMillis` | `DATABASE_POOL_IDLE_TIMEOUT_MS` | 30000          | The amount of time (in milliseconds) that a connection must be idle for before it is marked as a candidate for eviction.                                                                 |
-| `applicationName`        | `DATABASE_APPLICATION_NAME`     | `unleash`      | The name of the application that created this Client instance.                                                                                    |
-| `schema`        | `DATABASE_SCHEMA`     | `public`      | The schema to use in the database.                                                                                    |
-
-Alternatively, you can use a [libpq connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) to connect to the database. You can provide it directly or from a file by using one of the below options. In JavaScript, these are top-level properties of the root configuration object, *not* the `db` object.
-
-| Property name     | Environment variable | Default value | Description                                                                                                                                                                              |
-|-------------------|----------------------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `databaseUrl`     | `DATABASE_URL`       | N/A           | A string that matches the [libpq connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING), such as `postgres://USER:PASSWORD@HOST:PORT/DATABASE`. |
-| `databaseUrlFile` | `DATABASE_URL_FILE`  | N/A           | The path to a file that contains a [libpq connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING).                                               |
-
+| Property name | Environment variable | Default value | Description |
```

Comment:
Could keep the existing formatting, not sure if it's enforced though.

Reply:
That's odd, I guess intellij auto-applied the formatting.

Reply:
Interesting, it's not intellij, it's our lint-changed and eslint together making these changes. I'll leave it as is. provided it doesn't break the documentation. If it does, we'll need to make sure we filter md files.

Reply:
Oh right. After fixing `husky`, we now format changed `md` files with prettier. Can be changed in `package.json` if we want. 

/cc @thomasheartman

Reply:
I think it's fine. It shouldn't break the render in any way. This is probably a result of my Emacs, which formats md tables in such a way that they line up as plaintext. I'm happy to move to prettier's formatting instead, though, so I don't think we need to change it for tables.